src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/ActivatorUtilitiesTests.cs

```diff
@@ -171,25 +172,38 @@ public void CreateInstance_ClassWithABC_MultipleCtorsWithSameLength_ThrowsAmbigu
             Assert.Equal(message, exception.Message);
         }
 
-        [Fact]
-        public void CreateFactory_CreatesFactoryMethod()
+        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
```

Comment:
(nit) we are losing coverage here on platforms that don't support RemoteExecutor (WASM, Mobile, etc).

Maybe keep the existing test method and add a new single test that does the same thing when `IsDynamicCodeSupported=false`.